react/week3/src/components/button/button.js

```diff
@@ -0,0 +1,17 @@
+import "./button.css";
+
+function Button({ props, onClick }) {
+
+    const { name, value } = props;
+
+    const clicked = () => {
+        onClick(value);
+    }
+
+    return (
+        <button onClick={clicked.bind(this)} >{name}</button>
```

Comment:
It's not necessary to bind here.

You can simply pass the function as a reference:
`<button onClick={clicked} >{name}</button>`
..and when the user clicks on the button the function will be called.